### PR TITLE
Issue 4561 - Removed access to v4

### LINKED ISSUE #4561: Remove v4 front end routes
### Description
we've got the green light to decomission the v4 pages. Of course anything we've face lifted from v4 to v5 will still need to work on the v5 pages.

### Goals
- all v4 pages should be removed
- v5 pages with v4 components should still work as before

### Tasks
- [x] get rid of access to v5 route
- [x] get rid of isv5() function

### PR DESCRIPTION
**Pull request by sanmont3drepo**
This fixes #4561 

#### Description
- Removed access to v4.
- Removed isv5 conditional rendering
- Updated 404 now it only shows v5 version

### CODE REVIEW

**Comment by Amantini1997** [comment]
If I try to navigate to `https://3drepo.lan/dashboard`, I don't get redirected to v5 and the app crashes

File: frontend/src/main.tsx
Comment (Amantini1997) [inline frontend/src/main.tsx:1]: If I try to navigate to `https://3drepo.lan/dashboard` or `https://3drepo.lan/viewer`, I don't get redirected to v5 and the app crashes.

Also, if I insert an invalid route(e.g. `https://3drepo.lan/reset-password`), I don't get a 404

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

File: frontend/src/main.tsx
Comment (sanmont3drepo) [reply frontend/src/main.tsx:1]: cant reproduce

**Review by Amantini1997 — APPROVED** [review]

File: frontend/src/v4/routes/components/invitationDialog/invitationDialog.component.tsx
Comment (The-Daniel) [inline frontend/src/v4/routes/components/invitationDialog/invitationDialog.component.tsx:1]: Not sure if it's specifically from this file, but trying to invite a user now causes a crash

File: frontend/src/v4/constants/viewerGui.ts [hunk 1]
```diff
@@ -104,9 +103,7 @@ export const getViewerLeftPanels = () =>  [
 	VIEWER_PANELS.SEQUENCES,
 	VIEWER_PANELS.MEASUREMENTS,
 ].filter((panel) =>
-(clientConfigService.sequencesEnabled || panel !== VIEWER_PANELS.SEQUENCES)
-&& (isV5() || panel !== VIEWER_PANELS.TICKETS) // If is not in v5 dont add the tickets button
-).map(getPanelConfig);
+(clientConfigService.sequencesEnabled || panel !== VIEWER_PANELS.SEQUENCES)).map(getPanelConfig);
```
Comment (The-Daniel) [inline frontend/src/v4/constants/viewerGui.ts:106]: Opening the sequences panel is causing a crash

File: frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItemContainer.component.tsx [hunk 2]
```diff
@@ -57,7 +56,7 @@ export const GroupsTreeListItemComponent = (
 		<GroupsTreeListItemContainer $depth={depth}>
 			{children}
 			<ButtonsContainer>
-				{!isV5() && <TooltipButton action={onClickIsolate} label="Isolate" Icon={Visibility} disabled={disabled} />}
+				<TooltipButton action={onClickIsolate} label="Isolate" Icon={Visibility} disabled={disabled} />
```
Comment (The-Daniel) [inline frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItemContainer.component.tsx:59]: This should be removed as the isV5 is negated. This causes the styling to look weird.

image

(staging)
image

**Review by The-Daniel — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Pull request merged**